src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from "react";
+import { Icon, IconKey } from "src/components/Icon";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconButton } from "../IconButton";
+import { useToastContext } from "./ToastContext";
+
+export type ToastTypes = "error" | "warning" | "success" | "info" | "alert";
+export interface ToastProps {
+  type: ToastTypes;
+  message: ReactNode;
+}
+
+export function Toast(props: ToastProps) {
+  const { message, type } = props;
+  // using toast context to set notice to undefined once the user clicks out of the alert
+  const { setNotice } = useToastContext();
+  const tid = useTestIds(props, "toast");
+
+  return (
+    <div css={{ ...variantStyles[type], ...Css.df.aic.w100.gap2.px2.py2.$ }} {...tid} role="alert">
+      <span css={Css.fs0.plPx(12).$}>
```

Comment:
**thought/suggestion:** Based on the Figma mocks, we do not need the `plPx(12)` here. Appears to me the `px2` on the container would be enough padding.

Reply:
![Screen Shot 2022-11-09 at 1 50 02 PM](https://user-images.githubusercontent.com/1143861/200915855-df88c4ee-4fe3-45b3-9b6c-9b9d91267687.png)